security-analysis/security-analysis-api/src/main/java/com/powsybl/security/results/OperatorStrategyResult.java

```diff
@@ -7,46 +7,110 @@
+            this.networkResult = Objects.requireNonNull(networkResult);
+        }
 
-    private final NetworkResult networkResult;
+        public String getConditionalActionsId() {
+            return conditionalActionsId;
+        }
+
+        public PostContingencyComputationStatus getStatus() {
+            return status;
+        }
+
+        public LimitViolationsResult getLimitViolationsResult() {
+            return limitViolationsResult;
+        }
+
+        public NetworkResult getNetworkResult() {
+            return networkResult;
+        }
+    }
+
+    private final OperatorStrategy operatorStrategy;
+
+    private List<ConditionalActionsResult> conditionalActionsResults = new ArrayList<>();
 
     public OperatorStrategyResult(OperatorStrategy operatorStrategy, PostContingencyComputationStatus status, LimitViolationsResult limitViolationsResult,
                                   NetworkResult networkResult) {
         this.operatorStrategy = Objects.requireNonNull(operatorStrategy);
-        this.status = Objects.requireNonNull(status);
-        this.limitViolationsResult = Objects.requireNonNull(limitViolationsResult);
-        this.networkResult = Objects.requireNonNull(networkResult);
+        this.conditionalActionsResults.add(new ConditionalActionsResult(operatorStrategy.getId(), status, limitViolationsResult, networkResult));
     }
 
-    public OperatorStrategy getOperatorStrategy() {
-        return operatorStrategy;
+    public OperatorStrategyResult(OperatorStrategy operatorStrategy, List<ConditionalActionsResult> conditionalActionsResults) {
+        this.operatorStrategy = Objects.requireNonNull(operatorStrategy);
+        this.conditionalActionsResults = conditionalActionsResults;
     }
 
+    /**
+     * @return The PostContingencyComputationStatus of the final operator strategy result
+     */
     public PostContingencyComputationStatus getStatus() {
-        return status;
+        return getFinalOperatorStrategyResult().getStatus();
     }
 
+    /**
+     * @return The LimitViolationsResult of the final operator strategy result
+     */
     public LimitViolationsResult getLimitViolationsResult() {
-        return limitViolationsResult;
+        return getFinalOperatorStrategyResult().getLimitViolationsResult();
     }
 
+    /**
+     * @return The NetworkResult of the final operator strategy result
+     */
     public NetworkResult getNetworkResult() {
-        return networkResult;
+        return getFinalOperatorStrategyResult().getNetworkResult();
+    }
+
+    /**
+     * @return The Operator strategy associated to this result
+     */
+    public OperatorStrategy getOperatorStrategy() {
+        return operatorStrategy;
+    }
+
+    /**
+     * @return The list of ConditionalActionsResult for each conditional actions of the associated operator strategy
+     */
+    public List<ConditionalActionsResult> getConditionalActionsResult() {
```

Comment:
Why not `getConditionalActionsResults()` here? And a good method could be `getConditionalActionsResult(String id)`